core/docz-core/src/commands/dev.ts

```diff
@@ -44,7 +43,6 @@ export const dev = async (args: Arguments<any>) => {
 
   try {
     await dataServer.start()
-    if (args.open || args.o) openBrowser(`http://${config.host}:${config.port}`)
```

Comment:
This was a bad open anyway since it always just opened the index even if you configured a base path.